Formula/nuget.rb

```diff
@@ -1,8 +1,8 @@
 class Nuget < Formula
   desc "Package manager for Microsoft development platform including .NET"
   homepage "https://www.nuget.org/"
-  url "https://dist.nuget.org/win-x86-commandline/v5.4.0/nuget.exe" # make sure libexec.install below matches case
-  sha256 "880f8d306a65932b11f7edd7768b57d20e78fc618b21d785b303da27facc9a70"
+  url "https://dist.nuget.org/win-x86-commandline/v5.5.1/nuget.exe" # make sure libexec.install below matches case
```

Comment:
https://www.nuget.org/downloads lists `v5.4.0` as the recommended latest.

If you like JSON then you can also check https://dist.nuget.org/tools.json for `ReleasedAndBlessed`.

Reply:
Sounds good, I will update livecheck to reflect this information source. 👍

Reply:
I have to say, I really like `ReleasedAndBlessed` 😄